docs/serverless/alerts/alert-schema.mdx

```diff
@@ -904,4 +904,13 @@ The non-ECS fields listed below are beta and subject to change.
       
     </DocCell>
   </DocRow>
+    <DocRow>
+    <DocCell> `kibana.alert.intended_timestamp`</DocCell>
+    <DocCell>
+      Represents the estimated time when the alert was created by a manual rule run.
```

Comment:
This timestamp represents the time when the alert would have been created based on the source event timestamp.

Reply:
@approksiu how's this:

```suggestion
      Represents the time of when the alert would have been created based on the source event timestamp. This field is only populated for alerts generated by manual rule runs.
```

Reply:
Added to ESS docs via https://github.com/elastic/security-docs/pull/5631/files#r1773602370.